ci/run_tests/run_tests_windows.py

```diff
@@ -72,6 +72,14 @@ def main():
         default=[],
         help="Pcap++ tests to skip",
     )
+    parser.add_argument(
+        "--include-tests",
+        "-t",
+        type=str,
+        nargs="+",
+        default=[],
+        help="Pcap++ tests to include",
+    )
     parser.add_argument(
```

Comment:
This change was needed to support running only WinDivert tests in the `windivert` job in CI